core/src/main/java/com/linecorp/armeria/server/HttpResponseSubscriber.java

```diff
@@ -170,9 +170,18 @@ public void onNext(HttpObject o) {
                 }
 
                 final HttpHeaders additionalHeaders = reqCtx.additionalResponseHeaders();
+                final HttpHeaders additionalTrailers = reqCtx.additionalResponseTrailers();
+
                 final ResponseHeadersBuilder newHeaders = fillAdditionalHeaders(headers, additionalHeaders);
+                if (endOfStream && !additionalTrailers.isEmpty()) {
```

Comment:
Just curious. Would it be better to do this after [this code block](https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/server/HttpResponseSubscriber.java#L183-L201)? Because `endOfStream` flag can be updated in that code block.

Reply:
Ah thanks for noticing - moved to after that block

Reply:
Seems that the L172-175 can also be moved to here. :-)